debian/copyright

```diff
@@ -0,0 +1,147 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 2014-2019, Seagate Technology LLC and/or its Affiliates
+           2014-2019, Seagate Technology LLC and/or its Affiliates, br
+           \(co
+License: MPL-2
+Comment: No explicit license found, using license(s) from:
+ LICENSE.md
+
+Files: opensea-transport/*
+Copyright: 2012-2020, Seagate Technology LLC and/or its Affiliates
+License: MPL-2
+
+Files: opensea-operations/*
+Copyright: %" PRIu8 "\n", pwrTelData->temperatureCelcius
```

Comment:
Is this like this in upstream?

Reply:
It is not, seems like `decopy` did not parse it correctly. I'll fix it manually.